sdk/qrcode/android/java/com/google/cardboard/sdk/qrcode/CardboardParamsUtils.java

```diff
@@ -80,6 +80,25 @@
           .appendEncodedPath(URI_PATH_CARDBOARD_HOME)
           .build();
 
+  /**
+   * Converts a bytes array with an URI string to a Uri object.
+   * Obtains the physical parameters of a Cardboard headset from the Uri (as bytes) by
+   * calling {@code createFromUri}.
+   * Then writes the device parameters to a predefined storage location calling {@code
+   * writeDeviceParams()}.
+   *
+   * @param byteArrUri A bytes buffer with the URI to read the parameters from.
+   * @param context The current Context. It is generally an Activity instance or wraps one or an
+   *        Application. It is used to write to storage.
+   */
+  private static void saveParamsFromUri(byte[] byteArrUri, Context context){
```

Comment:
AFAICS, this is not consistent with the documentation as no redirections are supported. You'd need to do something pretty similar to: https://github.com/googlevr/cardboard/blob/4382d959a62196405ab75a237b5c39094a137c6e/sdk/qrcode/android/java/com/google/cardboard/sdk/qrcode/QrCodeContentProcessor.java#L155., since once the bytes array has been converted into an String and then into a Uri, exactly the same flow is required, PTAL.